tap_square/client.py

```diff
@@ -105,14 +107,36 @@ def get_locations(self):
         yield (result.body.get('locations', []), result.body.get('cursor'))
 
         while result.body.get('cursor'):
+            body['cursor'] = result.body.get('cursor')
```

Comment:
I don't see why to store this when you can just inline the variable in the result line in line number 112 below.